parsley/shared/src/main/scala/parsley/internal/deepembedding/frontend/PrimitiveEmbedding.scala

```diff
@@ -14,33 +14,57 @@ import parsley.internal.deepembedding.backend, backend.StrictParsley
 private [parsley] final class Attempt[A](p: LazyParsley[A]) extends Unary[A, A](p) {
     override def make(p: StrictParsley[A]): StrictParsley[A] = new backend.Attempt(p)
 
+    // $COVERAGE-OFF$
     override def visit[T, U[+_]](visitor: LazyParsleyIVisitor[T, U], context: T): U[A] = visitor.visit(this, context)(p)
+
+    override private[parsley] def prettyName = "attempt"
```

Comment:
`"atomic"`

Reply:
Renamed in [0b3309d](https://github.com/j-mie6/parsley/pull/219/commits/0b3309d533215e4f3094a4fc4019491971b00d9e)